README.md

```diff
@@ -5,6 +5,7 @@ puppeteer-screenshot-tester
 [![yarn](https://img.shields.io/badge/yarn-1.x-brightgreen.svg)]()
 
 Small library that allows us to compare screenshots generated by puppeteer in our tests.
+<br/>If no previous screenshot to compare is found, program will create one for you with current page screenshot.
```

Comment:
Usually first paragraph is just a description of library and doesn't contain any usage details